### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Include a content markup reference as a criterion for reviewing new documentation posts

## Issues resolved by this PR

N/A

### CODE REVIEW

**Comment by thlmenezes** [comment]
Inspired by comment https://devopness.atlassian.net/browse/DVN-11046?focusedCommentId=13386

note: using comment to avoid linking PR to the issue

File: docs/README.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ Devopness maintains its documentation in an open source repository, so feel free
 # Contributing
 Make sure all files created in this folder adhere to the following basic rules:
 1. Are defined as `Markdown` files (.md)
-2. Use front matter headers, see the list of available headers in [this topic](#predefined-variables)
-3. **DO NOT** set heading level one only one (`#`) in `Markdown` content, as it is reserved for the documentation article title. Headings inside the documentation article content must start from heading level two (`##`). Please refer to [Markdown basic syntax](https://www.markdownguide.org/basic-syntax/#headings) for examples and detailed instructions
-4. **DO NOT** use `HTML` tags in `Markdown` content
+1. Follow the [Github's Content Markup Reference](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md)
```
Comment (jfoliveira) [inline docs/README.md:7]: I believe we should be careful when pointing contributors to follow external guides that are not maintained by our team, neither has become an industry standard.
For example, what if a contributor adds the GitHub so called "tool tags" to our content?
```j2
{% webui %}
{% cli %}
{% codespaces %}
{% curl %}
{% jetbrains %}
```

This is described in the guide you are linking here, therefore the contributors will expect that to work with Devopness documentation, but it won't.

Given that, if you're really want to mention the external document as a reference, please be specific and mention the specific topic[s] that we can commit to follow and support.

File: docs/README.md [hunk 2]
```diff
@@ -4,9 +4,10 @@ Devopness maintains its documentation in an open source repository, so feel free
...
+1. Use front matter headers, see the list of available headers in [this topic](#predefined-variables)
+1. **DO NOT** set heading level one only one (`#`) in `Markdown` content, as it is reserved for the documentation article title. Headings inside the documentation article content must start from heading level two (`##`). Please refer to [Markdown basic syntax](https://www.markdownguide.org/basic-syntax/#headings) for examples and detailed instructions
```
Comment (jfoliveira) [inline docs/README.md:9]: Is this sentence correct? `one only one`?
```
1. **DO NOT** set heading level one only one
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (thlmenezes) [reply docs/README.md:9]: Yeah this sentence can be improved.

note: I've kept the original text, only changing the list numbering

File: docs/README.md (hunk 1 above)
Comment (thlmenezes) [reply docs/README.md:7]: You are correct, I'll make changes to only import the necessary recommendations from the external guide to accomplish the original goal.

**Pull request closed without merging**